index.php

```diff
@@ -634,7 +634,12 @@ public function extractDownload() {
 		$downloadedVersion = $this->getVersionByVersionFile(dirname($downloadedFilePath) . '/nextcloud/version.php');
 		$currentVersion = $this->getVersionByVersionFile($this->baseDir . '/../version.php');
 		if(version_compare($downloadedVersion, $currentVersion, '<')) {
-			throw new \Exception('Downloaded version is lower than installed version');
+			// If the version is not lower, only the same major and minor are allowed
```

Comment:
I dont get what the difference is, whether you download the files and put them on the server or the script did it. The version ignoring was not done for dailies, but for minor releases, to exactly allow this, so we don't have to do a version race and release a new version after owncloud so people could update again...